Libplanet.Tests/Store/Trie/KeyValueStoreTest.cs

```diff
@@ -67,7 +67,9 @@ public void Set()
         public void SetMany()
         {
             var values = new Dictionary<KeyBytes, byte[]>();
-            foreach (int i in Enumerable.Range(0, 10))
+#pragma warning disable SA1312
```

Comment:
@planetarium/libplanet  I think we need to tune stylecop.

Reply:
See the comment above about commenting pragma warnings. 😗